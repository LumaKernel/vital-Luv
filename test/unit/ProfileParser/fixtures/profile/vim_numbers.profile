SCRIPT  /test/test_numbers.vim
Sourced 1 time
Total time:   0.000075
 Self time:   0.000075

count  total (s)   self (s)
                            
    1              0.000004 123
    1              0.000002 123  456  789
                            

