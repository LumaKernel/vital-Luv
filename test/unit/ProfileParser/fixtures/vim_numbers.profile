SCRIPT  /home/luma/vim-make-plugin/vital-Luv/dev/ProfileParser/test_numbers.vim
Sourced 1 time
Total time:   0.000084
 Self time:   0.000084

count  total (s)   self (s)
                            
    1              0.000004 123
    1              0.000024 123  456  789
                            

