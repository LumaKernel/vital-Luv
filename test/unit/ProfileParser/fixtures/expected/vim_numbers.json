[{"path":"/test/test_numbers.vim","total_time":7.5e-5,"self_time":7.5e-5,"type":"script","lines":[{"content":""},{"self_time":4.0e-6,"count":1,"content":"123"},{"self_time":2.0e-6,"count":1,"content":"123  456  789"},{"content":""}],"count":1}]
